src/components/DeliverableView/DocumentsList.tsx

```diff
@@ -1,10 +1,59 @@
-import { Box } from '@mui/material';
+import { useState } from 'react';
+import { Box, Grid } from '@mui/material';
+import strings from 'src/strings';
+import { SearchSortOrder } from 'src/types/Search';
+import { useLocalization } from 'src/providers';
+import { TableColumnType } from 'src/components/common/table/types';
+import { BaseTable as Table } from 'src/components/common/table';
+import { SortOrder } from 'src/components/common/table/sort';
 import { ViewProps } from './types';
 
+const columns = (activeLocale: string | null): TableColumnType[] =>
+  activeLocale
+    ? [
+        {
+          key: 'name',
+          name: strings.DOCUMENT_NAME,
+          type: 'string',
+        },
+        {
+          key: 'description',
+          name: strings.DESCRIPTION,
+          type: 'string',
+        },
+        {
+          key: 'dateUploaded',
+          name: strings.DATE_UPLOADED,
+          type: 'string',
+        },
+      ]
+    : [];
+
 const DocumentsList = (props: ViewProps): JSX.Element => {
+  const { activeLocale } = useLocalization();
+
+  const [searchSortOrder, setSearchSortOrder] = useState<SearchSortOrder>({
+    field: 'deliverableName',
+    direction: 'Ascending',
+  } as SearchSortOrder);
+
+  const onSortChange = (order: SortOrder, orderBy: string) =>
+    setSearchSortOrder({
+      field: orderBy,
+      direction: order === 'asc' ? 'Ascending' : 'Descending',
+    });
+
   return (
     <Box display='flex' flexDirection='column'>
-      <div>DocumentsList coming soon {props.isAcceleratorConsole}!</div>
+      <Grid item xs={12}>
+        <Table
```

Comment:
if we go with OrderPreserved table, use an id that differentiates between isAccelerator vs not.

Reply:
Ah, I see, ignore my other comment.

Reply:
The participant / accelerator specific ID is going to be in #2214

Reply:
Got it, thanks @nickgraz 
Let's get this in and if you prefer to split them into scene specific views, let's do that. I'd prefer that.